node-fs-readfile/cat.js

```diff
@@ -0,0 +1,20 @@
+import { readFile } from 'node:fs/promises';
+
+async function cat() {
+  try {
+    const commandArray = process.argv.slice(2);
+    const promiseArray = commandArray.map(async (file) => {
+      const filePath = new URL(file, import.meta.url);
+      const contents = await readFile(filePath, { encoding: 'utf8' });
+      return contents;
+    });
```

Comment:
Although the Node documentation may look like you need import.meta.url, etc., it actually isn't necessary. By default ,readFile will look in the current directory for the file.

So, you can write this much more succinctly as:
`const promises = arrayOfFiles.map((file) => readFile(file, 'utf8'));`

(Note that as a shortcut, `readFile` lets you pass the encoding as a string in the 2nd parameter.)

Reply:
I have revised the readFiles to reflect on the comments, thank you!